tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -79,6 +79,13 @@ public static void Initialize()
                 }
 
                 Log.Information("IsProfilerAttached: true");
+
+                var asm = typeof(Instrumentation).Assembly;
+                Log.Information($"[Assembly metadata] Location: {asm.Location}");
```

Comment:
Should these be debug logs or did you want to keep them as info logs?

Reply:
I want these to show up everytime we load `Datadog.Trace`
They seem invaluable to me for troubleshooting.